crates/revive-strategy/src/cheatcodes/mod.rs

```diff
@@ -164,6 +167,11 @@ fn set_timestamp(new_timestamp: U256, ecx: Ecx<'_, '_, '_>) {
     });
 }
 
+fn set_basefee(new_basefee: U256, ecx: Ecx<'_, '_, '_>) {
```

Comment:
okay, so i assume it's not possible to modify the value of `Weights::seal_base_fee` on pallet-revive?

Reply:
and do we have anyone who's up to date on gas mapping?